### ISSUE TITLE
Write UI Tests for Signup screen

### ISSUE THREAD

**Issue by jogendra** [fossasia/open-event-orga-iOS#18]
**Expected Behaviour**
Write good UI Tests for the Login screen

**Screenshots of the issue**

N/A

**Would you like to work on the issue?**

Anyone can take up this issue

**Comment by jogendra** [comment]
![adp-fossasia-logo](https://user-images.githubusercontent.com/20956124/51316545-047b4d80-1a7b-11e9-9f35-99e3098a5f7b.png)

**Comment by dilumdesilva** [comment]
Hello @jogendra 

I would like to work on this issue

**Comment by jogendra** [comment]
@dilum1995 Yes, sure. first, you have to make login functionality work.

**Comment by dilumdesilva** [comment]
Okay I will work on that @jogendra

**Comment by ThaysPrado** [comment]
Still open? Can I help you with this?

**Comment by jogendra** [comment]
@ThaysPrado Yes, it is open and you can work on this. Looking forward to your PR :)
